diff --git a/engine/core/src/main/java/com/codingame/gameengine/core/GameManager.java b/engine/core/src/main/java/com/codingame/gameengine/core/GameManager.java
index 2a4d8051..70a66271 100644
--- a/engine/core/src/main/java/com/codingame/gameengine/core/GameManager.java
+++ b/engine/core/src/main/java/com/codingame/gameengine/core/GameManager.java
@@ -572,7 +572,8 @@ abstract public class GameManager<T extends AbstractPlayer> {
     private void addTurnTime() {
         totalTurnTime += turnMaxTime;
         if (totalTurnTime > GAME_DURATION_HARD_QUOTA) {
-            throw new RuntimeException(String.format("Total game duration too long (>%dms)", GAME_DURATION_HARD_QUOTA));
+            //throw new RuntimeException(String.format("Total game duration too long (>%dms)", GAME_DURATION_HARD_QUOTA));
+            log.warn(String.format("Total game duration too long (>%dms)", GAME_DURATION_HARD_QUOTA));
         } else if (totalTurnTime > GAME_DURATION_SOFT_QUOTA) {
             log.warn(
                 String.format(
diff --git a/pom.xml b/pom.xml
index 1318a640..93ef36d7 100644
--- a/pom.xml
+++ b/pom.xml
@@ -96,6 +96,16 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+
+			<!--
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-gpg-plugin</artifactId>
@@ -110,6 +120,7 @@
 					</execution>
 				</executions>
 			</plugin>
+			-->
 		</plugins>
 	</build>
 </project>
